app/helpers/publications_helper.rb

```diff
@@ -14,9 +14,9 @@ def content_tagger_url(edition)
     Plek.new.external_url_for("content-tagger") + "/content/#{content_id}"
   end
 
-  def enabled_users_select_options(empty_value = true)
+  def enabled_users_select_options
     options = User.enabled.order_by([%i[name asc]]).collect { |u| [u.name, u.id] }
-    options.unshift(["", ""]) if empty_value
+    options.unshift(["", ""])
     options
```

Comment:
We can probably remove this line now, as `unshift` returns the array I believe.

Reply:
Oh yes - nice catch. Just pushed up a fix.